doc/Requirements Specification/Parts/Purpose.lyx

```diff
@@ -677,7 +691,20 @@ glsuseriv{software architecture}
 
 \end_inset
 
- from their source code.
+ from their source code, as this is done by 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+gls{SoMoX}
```

Comment:
I’d prefer “tools like SoMoX” as this could theoretically be done by other tools, too.